contracts/stabilizer/EthPegStabilityModule.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./PegStabilityModule.sol";
+
+/// @notice ETH PSM that allows separate pausing of mint and redeem
+/// by the guardian and governor
+contract EthPegStabilityModule is PegStabilityModule {
+
+    /// @notice boolean switch that indicates whether redemptions are paused
+    bool public redeemPaused;
+
+    /// @notice event that is emitted when redemptions are paused
+    event RedemptionsPaused(address account);
+    
+    /// @notice event that is emitted when redemptions are unpaused
+    event RedemptionsUnpaused(address account);
```

Comment:
I know it changes the base class but I think these should be canon for all PSMs not just ETH

Reply:
it feels like the scope keeps changing here.

Reply:
if we do this for all base psms, then they should have separate pause switches for mint and redeem outside of oz

Reply:
that would mean we have 3 separate pieces of code doing pausing and at that point we should just move to function selector pausing.

Reply:
Don't feel strongly. ETH can be unique